lib/builder/builder.js

```diff
@@ -531,18 +514,19 @@ export default class Builder {
   }
 
   watchFiles() {
+    const src = this.options.srcDir
     const patterns = [
-      r(this.options.srcDir, 'layouts'),
-      r(this.options.srcDir, 'store'),
-      r(this.options.srcDir, 'middleware'),
-      r(this.options.srcDir, 'layouts/*.vue'),
-      r(this.options.srcDir, 'layouts/**/*.vue')
+      r(src, 'layouts'),
+      r(src, 'store'),
+      r(src, 'middleware'),
+      r(src, 'layouts/*.vue'),
+      r(src, 'layouts/**/*.vue')
     ]
-    if (this._nuxtPages) {
-      patterns.push(r(this.options.srcDir, 'pages'))
-      patterns.push(r(this.options.srcDir, 'pages/*.vue'))
-      patterns.push(r(this.options.srcDir, 'pages/**/*.vue'))
-    }
+    this._nuxtPages && patterns.push(
```

Comment:
What's your idea if we keep using `ifs` for more consistency? I've seen some linters throwing errors about such usage.